src/openrct2/core/FileScanner.cpp

```diff
@@ -67,7 +67,7 @@ class FileScannerBase : public IFileScanner
     {
         std::string                 Path;
         std::vector<DirectoryChild> Listing;
-        sint32                      Index;
+        sint32                      Index{};
```

Comment:
Can you put a space between `Index` and `{}`? (This applies to all occurrences, but I'll say this once.)

Reply:
It doesn't really make sense to initialize a type with `{}` to me, I'd rather see `= 0` here.